tests/platform_helper/test_command_environment.py

```diff
@@ -619,6 +619,7 @@ class TestGenerate:
         return_value=(["def456"], ["ghi789"]),
     )
     @patch("dbt_platform_helper.commands.environment.get_vpc_id", return_value="vpc-abc123")
+    @patch("dbt_platform_helper.utils.validation.get_aws_session_or_abort")
     @patch("dbt_platform_helper.commands.environment.get_aws_session_or_abort")
```

Comment:
Is this double import of `get_aws_session_or_abort` just to make sure https://github.com/uktrade/platform-tools/blob/a92105bf65edc740eace21ff4ef27bdbf436856b/dbt_platform_helper/utils/validation.py#L469 works? Just a curiosity question

Other than that the PR is 👍🏻

Reply:
Yes - because the function is used in two places, once in validation and once in the command directly, you need to patch it in both places